paper/main.tex

```diff
@@ -679,20 +677,19 @@ \section{Illustrative Example}\label{sec:illustrative-example}
 ability to save and load pre-trained models is not guaranteed across
 different \texttt{scikit-learn} versions.  For example, suppose that
 the \texttt{Pipeline} object was created using \texttt{scikit-learn}
-v0.21.3).  In that version of \texttt{scikit-learn}, the
-\texttt{LatentDirichletAllocation} class is defined in
-\texttt{sklearn.decomposition.\_lda}.  However, in version
+v0.21.3.  In that version of \texttt{scikit-learn}, the
+\texttt{LatentDirichletAllocation} class was defined in
+\texttt{sklearn.decomposition.online\_lda}.  However, in version
 0.22.0, that module was renamed to \texttt{\_online\_lda}, and in
-versions 0.22.1 and higher, the name was reverted back to
-\texttt{\_lda}.
+version 0.22.1, it was again renamed to \texttt{\_lda}.
```

Comment:
What was the difference between v0.21.3 and 0.22.0?

Seems like .online_lda --> .online_lda --> ._lda.  Maybe a typo?

Reply:
it goes `online_lda` --> `_online_lda` (with leading underscore) --> `_lda`

Reply:
got it :+1: